pages/wishlist.css

```diff
@@ -0,0 +1,45 @@
+:root {
+    --white: #f5f1f1;
+    --grey: #C4C4C480;
+    --red: #dc143c;
+}
+
+.wishlist-container {
+    width: 100vw;
```

Comment:
You can use 100% width as 100vw will give a horizontal scrollbar to your viewport

Reply:
thanks for your suggestion. i will change that.